### PR TITLE
WebPublisher: Fix for proper parsing of ffmpeg output arguments

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Batch names containing `-af` or `-vf` were triggering failure of `ExtractReview` with 
```At least one output file must be specified```

## Description
There is logic which handles presence of audio/video filters in output arguments for ffmpeg to yank them into proper position. They were just simple presence filters which hit on an issue that output arguments for files coming from WP contain batch name with `-` characters. 

## Additional info
In specific cases batch name could contain `-af` which broke parsing and produces command like 
```
"C:\Program Files (x86)\OpenPype\vendor\bin\ffmpeg\windows\bin\ffmpeg" -apply_trc gamma22 -i W:\4c9c1954-d342-44f6-962d-af347c59c83b\be9709f0-b370-47c5-bbde-f973c48464bb\wsb_sh060_2d_rough_b_v001.mp4 -filter:a "W:\4c9c1954-d342-44f6-962d347c59c83b\be9709f0-b370-47c5-bbde-f973c48464bb\wsb_sh060_2d_rough_b_v001_h264.mp4" -pix_fmt yuv420p -b:v 40M -minrate: 40M -maxrate: 40M -bufsize: 40M -intra -y
```
(See missing value at the end after -y)

## Testing notes:
1. Reprocess any batch that contains `-af` in its name
2. Configure `ExtractReview` to contain some audio/video filters to make sure they are correctly propagated
Screenshot 2022-02-15 152425

### CODE REVIEW

File: openpype/plugins/publish/extract_review.py [hunk 1]
```diff
@@ -694,13 +694,13 @@ def ffmpeg_full_args(
         audio_args_dentifiers = ["-af", "-filter:a"]
         for arg in tuple(output_args):
             for identifier in video_args_dentifiers:
-                if identifier in arg:
+                if arg.startswith("{} ".format(identifier)):
```
Comment (iLLiCiTiT) [inline openpype/plugins/publish/extract_review.py:697]:
```suggestion
                if arg.startswith("{} ".format(identifier)) or arg == identifier:
```

File: openpype/plugins/publish/extract_review.py [hunk 2]
```diff
@@ -694,13 +694,13 @@ def ffmpeg_full_args(
...
                     output_args.remove(arg)
                     arg = arg.replace(identifier, "").strip()
                     video_filters.append(arg)
 
             for identifier in audio_args_dentifiers:
-                if identifier in arg:
+                if arg.startswith("{} ".format(identifier)):
```
Comment (iLLiCiTiT) [inline openpype/plugins/publish/extract_review.py:703]:
```suggestion
                if arg.startswith("{} ".format(identifier)) or arg == identifier:
```

File: tests/unit/openpype/plugins/publish/test_extract_review.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from openpype.plugins.publish.extract_review import ExtractReview
```
Comment (iLLiCiTiT) [inline tests/unit/openpype/plugins/publish/test_extract_review.py:1]: I don't think we should put tests into plugins directories.

Comment (iLLiCiTiT) [reply tests/unit/openpype/plugins/publish/test_extract_review.py:1]: Oh it's not in plugins...

**Review by iLLiCiTiT — APPROVED** [review]

**Review by 64qam — APPROVED** [review]

**Pull request merged**